external/webserver/http_client.c

```diff
@@ -1263,6 +1263,11 @@ int http_send_response(struct http_client_t *client, int status, const char *bod
 
 int http_send_response_with_status_msg(struct http_client_t *client, int status, const char* status_message, const char* body, struct http_keyvalue_list_t *headers)
 {
+	if (body == NULL) {
```

Comment:
please check helper function called at end is accepting null body. 
You may also skip strlen and proceed. check if possible